### ISSUE TITLE
bug(rln-relay):  rln relay fails to mount if rln credential path is defined

### ISSUE THREAD

**Issue by rymnc** [status-im/nwaku#1202]
### Problem
mounting of the rln relay fails with no description while passing in the path to the credentials file

### Impact
medium

### To reproduce
If you can reproduce the behavior, steps to reproduce:

```sh
make chat2 RLN=true ONCHAIN_RLN=true
 ./build/chat2 --fleet:test --content-topic:/toy-chat/2/luzhou/proto --rln-relay:true --rln-relay-dynamic:true --eth-mem-contract-address:0x4252105670fe33d2947e8ead304969849e64f2a6 --eth-account-address:<address> --eth-account-privatekey:<pk> --eth-client-address:wss://goerli.infura.io/ws/v3/xxxx --rln-relay-cred-path="../../"
```

### Expected behavior
The rln relay should accept the credential path, import and use that to generate proofs to attach to RateLimitProof's

### Screenshots/logs

```
Choose a nickname >> p12
Welcome, p12!
Connecting to test fleet using DNS discovery...
Discovered and connecting to @[16Uiu2HAkvWiyFsgRhuJEb9JfjYxEkoHLgnUQmr1N5mKWnYjxYRVm, 16Uiu2HAmJb2e28qLXxT5kZxVUUoJt72EMzNGXB47Rxx5hw3q4YjS, 16Uiu2HAmPLe7Mzm8TsYUubgCAW1aJoeFScxrLj8ppHFivPo97bUZ]
Listening on
 /ip4/xxx/tcp/60000/p2p/16Uiu2HAmAyJpLWxZN4sxfPXzYbhqfJhPGgqmX4VQotg7wdEpTn9y
Store enabled, but no store nodes configured. Choosing one at random from discovered peers
Connecting to storenode: 16Uiu2HAmJb2e28qLXxT5kZxVUUoJt72EMzNGXB47Rxx5hw3q4YjS
rln-relay preparation is in progress...
failed to mount rln-relay: 
```

### nwaku version/commit hash

version / git commit hash: `v0.11-104-g107ca9`

**Comment by staheri14** [comment]
Can you please try it without wrapping the path inside `"`?

**Comment by staheri14** [comment]
I mean `--rln-relay-cred-path=../../`

**Comment by rymnc** [comment]
I believe https://github.com/status-im/nwaku/pull/1210 has resolved this bug. Need to test again

**Comment by rymnc** [comment]
Fixed by #1210
